games-roguelike/FTL-gog/FTL-gog-1.6.9.25330.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Can 7 be used here? I know these eclasses namely support it, but does it work?

Reply:
I didn't know 7 was available. I can try it out.